model-optimizer/extensions/middle/InterpolateSequenceToInterpolate.py

```diff
@@ -127,37 +230,54 @@ def replace_sequence(seq: List[Node], graph: Graph):
 
     dims_and_scales_ = []
     # Each element of the list dims_and_scales_ is a pair
-    #      (axis, output size for this axis)
-    for interp in seq:
-        dims_and_scales_.extend(zip(interp.axes, interp.in_port(1).get_connection().get_source().node.value))
-
-    axis_to_size = sorted(list(dict(dims_and_scales_).items()), key=lambda x: x[0])
-    axes_of_node = int64_array([z[0] for z in axis_to_size])
-    sizes = int64_array([z[1] for z in axis_to_size])
+    #      (axis, output size for this axis) (opset1)
+    # or
+    #      (axis, output size for this axis, output scales for this axis) (opset4)
+    if seq[0].get_opset() == 'opset1':
+        for interp in seq:
+            dims_and_scales_.extend(zip(Interpolate.get_axes(interp),
+                                        interp.in_port(1).get_connection().get_source().node.value))
```

Comment:
Why do you get the value like this: `interp.in_port(1).get_connection().get_source().node.value`? The attribute `value` may not be set for node. It could be dynamically calculated. You should use `interp.in_port(1).get_connection().get_source().data.get_value()`.

The same comment for lines 248-249.

Reply:
Done.